### DISCUSSION TITLE
Is fief cloud production ready?

### DISCUSSION THREAD

**Discussion by michaldev** [fief-dev/fief#127] [Q&A] [answered]
I am aware of the beta status of the project. Still, I'd like more details.

I have a project that I will be implementing in production in a few days. Hobbyist, but for a few real clients who will use it professionally. Initially, I assumed fief as something "for the moment" for prototyping. Currently I am faced with a dilemma - stay with fief or switch to Auth0.

1. Will the cloud be maintained? (migrations, continuity, etc.)
2. Will the prices be lower, higher or comparable to auth0?
3. Is there still a lot of changes waiting for fief that can undermine the foundations? Or will it just be more stable?
4. When will translations be available? (those from poeditor)

**Comment by frankie567** [comment, marked as answer]
Hi @michaldev 👋 

Thank you for seriously considering Fief for your project! To answer your questions:

1. Yes, the Cloud will be maintainted. That's how we plan to earn money and sustain our development.
2. Pricing is not fixed yet but what I can say is this:
    * It won't be a pricing per user (like 0.20$ / user) but fixed monthly plans including a number of users (e.g. 100 / 1000 / 10K / unlimited)
    * All features will be included no matter the plan, including advanced ones like Bring Your Own Database
3. Fief's foundation are quite stable and won't change in a breaking way. What we want is to add more features (more customization, 2FA, etc), but they shouldn't not break the existing parts.
4. Translations are updated on each release. I wanted to do a release in the coming days, so Polish will be included in this one

Hope that clarifies things 😃 Feel free to ask any more questions you might have!

**Reply by michaldev** [reply]
"including a number of users (e.g. 100 / 1000 / 10K / unlimited)" - Total users or monthly active users?

**Reply by frankie567** [reply]
From what I plan now, total users 🙂

**Reply by michaldev** [reply]
Ok, thanks a lot for your all replies! 🙂
